packages/combo-box/test/scrolling.test.js

```diff
@@ -1,64 +0,0 @@
-import { expect } from '@esm-bundle/chai';
-import { fixtureSync, focusout, isIOS } from '@vaadin/testing-helpers';
-import './not-animated-styles.js';
-import '../vaadin-combo-box.js';
-
-describe('scrolling', () => {
-  let comboBox, overlay, scroller, input;
-
-  beforeEach(() => {
-    comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
-    overlay = comboBox.$.overlay;
-    input = comboBox.inputElement;
-    scroller = comboBox._scroller;
-  });
-
-  afterEach(() => {
-    comboBox.close();
-  });
-
-  (isIOS ? describe : describe.skip)('iOS', () => {
-    it('should have momentum scrolling enabled', () => {
-      comboBox.open();
-
-      expect(getComputedStyle(scroller).WebkitOverflowScrolling).to.equal('touch');
-    });
-  });
-
-  describe('scrolling position', () => {
-    beforeEach(() => {
-      const items = [];
-
-      for (let i = 0; i < 100; i++) {
-        items.push(i.toString());
-      }
-
-      comboBox.items = items;
-    });
-
-    it('should be zero when no items are selected', () => {
-      comboBox.open();
-
-      expect(scroller.scrollTop).to.equal(0);
-    });
-
-    it('should be zero when the first item is selected', () => {
-      comboBox.value = comboBox.items[0];
-      comboBox.open();
-
-      expect(scroller.scrollTop).to.equal(0);
-    });
-
-    it('should not close the items when touching scroll bar', () => {
-      comboBox.open();
-      focusout(input, overlay);
-      expect(comboBox.opened).to.be.true;
-    });
-
-    it('should keep the focused attribute while scrolling', () => {
-      comboBox.open();
-      focusout(input, overlay);
-      expect(comboBox.hasAttribute('focused')).to.be.true;
-    });
```

Comment:
Moving these two remaining tests to `toggling-dropdown.test.js` suite which seems a proper place.